styles.css

```
@@ -13,19 +13,370 @@
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto:wght@300;400;700&display=swap');
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    /*border: 1px solid red;*/
+}
+
+
+/*Kleuren*/
+
+:root {
+    --color-purple: #5722CD;
+    --color-blue: #4845E4; /*header background*/
+    --color-pink: #ED4385;
+    --color-black-standard-text: #0C0C0C; /*standaard tekst*/
+    --color-light-purple-big-floating-words: #F2F0FF;
+    --color-light-grey-background-projects: #F3F6F9;
+    --color-grey-service-tile: #7A7A7A;
+}
+
 
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+body {
+    font-family: 'Roboto', Arial, sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', Arial, serif;
+}
+
+h1, h2 {
+    font-size: 2rem;
+    padding: 2rem 0;
+}
+
+h3 {
+    font-size: 1.5rem;
+    padding: 1.5rem 0;
+}
+
+h4 {
+    font-size: 1.3rem;
+    padding: 1.2rem 0;
+}
+
+h5 {
+    font-size: 1.1rem;
+    padding-top: 0.5rem;
+}
+
+p {
+    font-size: 1rem;
+    padding: 1rem 0;
+}
+
+
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    max-width: 100%;
+    justify-content: center;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    flex: 0 1 1200px; /*1200px gepakt volgens deze website https://van-ons.nl/blog/visueel-ontwerp/specificaties-van-een-goed-webdesign/*/
+    padding: 1rem;
+    align-items: center;
+    text-align: center; /* tekst centreeren binnen de container*/
+}
 
 /* --------------- 3.2 Other --------------------------- */
 
+button {
+    background-color: var(--color-pink);
+    border-radius: 1.5rem;
+    border: none;
+    padding: 1rem 2rem;
+    margin: 1rem;
+    color: white;
+    text-transform: uppercase;
+    box-shadow: 0 0.3rem 0.5rem 0.3rem rgba(0, 0, 0, .2);
+}
+
+button:hover {
+    cursor: pointer;
+}
+
+a {
+    text-decoration: none;
+}
+
+img {
+    width: 100%;
+}
+
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
+.header {
+    background-color: var(--color-blue);
+    background: linear-gradient(129deg, #4B49E8 0%, #4B49E8 26%, #5722CD 100%);
+    color: white;
+    position: relative;
+}
+
+.nav-bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+}
+
+.nav-bar > ul {
+    display: flex;
+    flex-direction: row;
+    flex: 0 1 400px;
+    justify-content: space-between;
+    padding: 0.5rem 0;
+    list-style-type: none;
+}
+
+.header-content-styling-div {
+    padding: 2rem 0 0 0;
```

Als je geen padding-right, -bottom en -left hebt om te overschrijven, dan prefereer ik `padding-top`
```suggestion
    padding-top: 2rem;
```